core/domain/user_jobs_one_off.py

```diff
@@ -243,9 +244,13 @@ def map(item):
         stories_linked_to_completed_explorations = (
             story_fetchers.get_stories_by_ids(
                 story_ids_linked_to_completed_explorations))
+        topic_ids_linked_to_stories = [
+            story.corresponding_topic_id for story in stories_linked_to_completed_explorations] # pylint: disable=line-too-long
```

Comment:
Don't use the pragma. Break before "for" or after "in".

Reply:
Done.